modules/raft/src/test/java/org/apache/ignite/raft/jraft/storage/impl/BaseLogStorageTest.java

```diff
@@ -16,6 +16,12 @@
  */
 package org.apache.ignite.raft.jraft.storage.impl;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Are you sure that imports should be here in jraft?

Reply:
Why not?

Reply:
I mean, we are trying to change as little as we can in jraft sources and this looks like an accidental import reformat

Reply:
Oh, thanks, good catch. I had no idea JRaft code has a different order of imports.